napari/components/viewer_model.py

```diff
@@ -616,9 +617,9 @@ def add_points(
             broadcastable to the same shape as the data.
         edge_width : float
             Width of the symbol edge in pixels.
-        edge_color : str
+        edge_color : str, array-like, vispy.color.Color, vispy.color.ColorArray
```

Comment:
still have to remove `vispy.color.Color` and `vispy.color.ColorArray` from all public doc strings - I won't flag them all